csFastFloat/Utils/Utils.cs

```diff
@@ -213,5 +217,64 @@ internal unsafe static bool strncasecmp(byte* input1, ReadOnlySpan<byte> input2,
 
     
 
+    [MethodImpl(MethodImplOptions.AggressiveInlining)]
+    public static int LeadingZeroCount(ulong value)
+    {
+#if HAS_BITOPERATIONS
+      return System.Numerics.BitOperations.LeadingZeroCount(value);
+#else
+      uint hi = (uint)(value >> 32);
+ 
+      if (hi == 0)
+      {
+        return 32 + Log2SoftwareFallback((uint)value);
+      }
+ 
+      return Log2SoftwareFallback(hi);
+      
+      [MethodImpl(MethodImplOptions.AggressiveInlining)]
+      static int Log2SoftwareFallback(uint value)
+      {
+       if (value == 0)         return 32;
```

Comment:
Before the next release, the code should get reformatted, here it appears that the indentation is changes from line to line.